core/src/main/java/com/dooapp/fxform/view/skin/FXMLSkin.java

```diff
@@ -60,6 +60,7 @@ public Node call() throws Exception {
     protected Node loadFxml() throws NodeCreationException {
         fxmlLoader = new FXMLLoader();
         fxmlLoader.setLocation(url);
+		fxmlLoader.setResources(getSkinnable().getResourceBundle());

```

Comment:
Small indentation issue here

Reply:
OK, this is because y set in my eclipse replaces indentation spaces with tabs, sorry for that.